lib/FunctionIsolation/EnforceABI.cpp

```diff
@@ -343,13 +328,9 @@ void EnforceABIImpl::handleRegularFunctionCall(CallInst *Call) {
 
   // Generate the call
   IRBuilder<> Builder(Call);
-  CallInst *NewCall = generateCall(Builder, Callee, Block, *CallSite);
```

Comment:
This is an erroneous removal of the call to the isolated function, therefore they disappear from the IR and therefore in the decompiled code

Reply:
gotcha!

Reply:
Undraft and ping me once this is fixed.